user-guide/Advanced_Functionality/Swarming/SwarmingFAQ.md

```diff
@@ -0,0 +1,71 @@
+---
+uid: SwarmingFAQ
+---
+
+# Frequently asked questions
+
+## Can I swarm X/Y/Z?
+
+Swarming will gradually become available for more and more functionality in DataMiner.
+
+As of DataMiner 10.5.1, only basic elements are supported.
+
+For a list of upcoming features, see [Upcoming features](xref:Swarming#upcoming-features).
+
+## Does the ID of an element change when it is swarmed?
+
+No, the identifier (DataMiner ID/element ID) of an element that is swarmed stays the same.
+
+Each element also stores where it is hosted, and that "Host ID" field is updated when the element is swarmed.
+
+## What does the **DataMinerID** mean of an element that swarmed?
+
+The meaning of a **DataMinerID** has slightly changed in a DMS with Swarming.
+
+It used to be that every element has an **ElementID** that uniquely identifies it on a DataMiner agent.
+However, once clusters of agents were created, these were no longer unique in the whole DMS, so the **DataMinerID** was added to the element identifier, which now consists of 2 number often shown as **DataMinerID/ElementID**.
+
+You can see this identifier when you right click your element and open *properties*.
+
+![What does the DataMinerID of an element mean](~/user-guide/images/Swarming_FAQ_DataMinerID.png)
+
+To find the agent that was hosting the element, that same **DataMinerID** was used.
+However this no longer works when swarming the element.
+To keep the identifier unique in the DMS this **DataMinerID** cannot change.
+
+To see the agent hosting the element in DataMiner Cube, you can click a parent view and scroll to the right until you see the **HOST ID** and **DATAMINER** column for the element.
+
+Example below (with columns reordered for convenience):
+
+![Hosting Agent in Cube](~/user-guide/images/Swarming_FAQ_HostingAgentCube.png)
+
+To figure out the agent hosting the element in code, you can use the **HostingAgentID** property instead.
+
+## Can I block an element from being swarmed?
+
+Not in the initial release but is expected to be available in DataMiner 10.5.5/10.6.0.
```

Comment:
Is it certain that this feature will be available in DataMiner 10.5.5/10.6.0? If not, it might be better if we don't mention the version yet, as we may forget to update it when this changes. Or if it is certain, to make this more future-proof, I'd adjust it to:
```suggestion
This feature is available from DataMiner 10.5.5/10.6.0 onwards.
```

Reply:
The feature is ready but still going through validation, so not officially available yet in RC.
I do expect it to be in time for the seal next week, but we can leave it out for now and we'll update it once we merged it in. That we there's no real harm if we forget.